apis/cloudidentity/v1beta1/types.generated.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+// +kcc:proto=mockgcp.cloud.cloudidentity.groups.v1beta1.ExpiryDetail
+type ExpiryDetail struct {
+	// The time at which the `MembershipRole` will expire.
+	// +kcc:proto:field=mockgcp.cloud.cloudidentity.groups.v1beta1.ExpiryDetail.expire_time
+	ExpireTime *string `json:"expireTime,omitempty"`
+}
+
+// +kcc:proto=mockgcp.cloud.cloudidentity.groups.v1beta1.Membership
+type Membership struct {
```

Comment:
Could you rerun the types generation command to clean up this file? We should add to the guide that after manually editing the types, the command needs to be rerun to update the generated code.

Reply:
did you mean this command:
```
cd dev/tools/controllerbuilder ; go run . generate-types      --service mockgcp.cloud.cloudidentity.groups.v1beta1      --resource CloudIdentityMembership:Membership      --api-version 'cloudidentity.cnrm.cloud.google.com/v1beta1'
```

Reply:
done.